src/parser/__tests__/Root.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 
-import Root from './Root'
+import Root from '../../Root'
```

Comment:
this looks like it ended up in the wrong test folder

Reply:
:notlikeblob: